swagger.yaml

```diff
@@ -2672,6 +2682,9 @@ definitions:
         type: string
         description: A comma-separated list of destination domain names, domains, IP addresses, or other network CIDRs to exclude from proxying.
         x-nullable: true
+      operators:
+        $ref: '#/definitions/operators'
```

Comment:
By default objects will not be part of a table unless to embed them, you can see an example here https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L2946

Reply:
@filanov  , what's the significance of `x-go-custom-tag` ?

Reply:
It's a way to tell the ORM to set specific configuration for the filed or the object. 
in this case it will tell the ORM to create a filed for each entry in the object and set it as part of the upper level object. 
I'm not sure how it will react to a nested object, so you should check it.
Connect to the DB and take a look at how is the table defined.